7_react/react1/week2/src/TodoList/TodoList.js

```diff
@@ -0,0 +1,53 @@
+import TodoItem from "TodoItem/TodoItem"
+import "./TodoList.css"
+
+
+const todoList = [{
+    num: 1,
+    description: "Get out of bed",
+    deadline: "Wed Sep 13 2017"
+},
+{
+    num: 2,
+    description: "Brush teeth",
+    deadline: "Thu Sep 14 2017"
+},
+{
+    num: 3,
+    description: "Eat breakfast",
+    deadline: "Fri Sep 15 2017"
+}]
+
+function TodoList(){
+    const todoItemsComponents = todoList.map(item => (
+        <TodoItem key={item.num} description={item.description} deadline={item.deadline}/>
+    ))
+        function App() {
+        const [name,setName] = useState("");
+        var myArray = [];
```

Comment:
Two things: 

1) It's good practice to be more specific with variable names. Remember that in the real world, there will be other people (your teammates etc) who will need to read and understand your code easily. Here, someone might ask "what is this array made of", "what elements are in the array" etc.

2) Store this array as a state. Otherwise, line 27 will be called every time the component re-renders, and its value will be reset, which is not what we want.